content/java/concepts/math-methods/terms/subtract-exact/subtract-exact.md

```diff
@@ -0,0 +1,48 @@
+---
+Title: '.subtractExact()'
+Description: 'Returns the difference of the arguments.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Methods'
+  - 'Numbers'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`Math.subtractExact()`** method returns the difference of the arguments.
```

Comment:
Add an explanation of the difference or benefit of using this function as opposed to the subtraction operator (-).

Reply:
I added an explanation to the best of my knowledge. Please suggest improvements.